scripts/create-release.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+set -eu -o pipefail
```

Comment:
```suggestion
```

It would be good if this was a package/output in the flake, that way you could use `pkgs.writeShellApplication` to ensure it gets run through `shellcheck` and all the dependencies of the script are provided

Reply:
Ahh yes, this also applies the `-e` and `pipefail` options automatically.

It's now available at `nix run .#create-release`